src/components/shows/showviewer.module.css

```diff
@@ -0,0 +1,189 @@
+a {
+  color: red;
+}
+
+.aboutImage {
+  object-fit: contain;
+  object-position: center;
+  width: 100%;
+  height: auto;
+}
+
+.aboutImageContainer {
+  margin-top: 30px;
+  margin-left: 48px;
+  display: flex;
+  flex: 0 0 33%;
+}
+
+.aboutText {
+  display: flex;
+  flex-direction: row;
+}
+
+.castImageContainer {
+  display: flex;
+  flex: 0 0 50%;
+}
+
+.castText {
+  margin-left: 24px;
+}
+
+.castText2 {
+  margin-left: 0;
+}
+
+.catchPhrase {
+  font-style: italic;
+  text-align: center;
+  vertical-align: center;
+  margin-bottom: 0px;
+  margin-top: 20px;
+  font-size: large;
```

Comment:
Font sizes should ideally be set using standard elements like `<h1> <h2>` etc. This keeps font sizes consistent across pages. You should still use this class to apply any extra formatting like italics though.